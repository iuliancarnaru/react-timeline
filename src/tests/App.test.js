import React from "react";
import Enzyme, { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";
import Adapter from "enzyme-adapter-react-16";
import App from "../components/App";

Enzyme.configure({ adapter: new Adapter() });

describe("<App /> component", () => {
  it("should render succesfuly", () => {
    const wrapper = shallow(<App />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
