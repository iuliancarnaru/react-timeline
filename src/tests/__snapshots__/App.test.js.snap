// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> component should render succesfuly 1`] = `
<div>
  <h1>
    CRUD with HOOKS
  </h1>
</div>
`;
